permissions/plugin-declarative-pipeline-migration-assistant.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: "declarative-pipeline-migration-assistant-plugin"
+github: "jenkinsci/declarative-pipeline-migration-assistant-plugin"
+paths:
+  - "org/jenkins-ci/plugins/to-declarative/declarative-pipeline-migration-assistant-plugin"
```

Comment:
We don't do `-plugin` suffixes anymore. Please remove this.

Reply:
Doh, missed that :-(